src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -65,6 +67,13 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
         self.STR_DPKG_WAS_INTERRUPTED = "E: dpkg was interrupted, you must manually run 'sudo dpkg --configure -a' to correct the problem."
         self.ESM_MARKER = "The following packages could receive security updates with UA Infra: ESM service enabled:"
 
+        # Ubuntu Pro Client pre-requisite checks.
+
```

Comment:
nit: remove line between comment and code. Empty space here suggests that there is code gone missing when that's not the case.

Reply:
Got it. Will make changes.